core/net/src/hybrid/rest_api.rs

```diff
@@ -0,0 +1,20 @@
+use actix_web::{HttpResponse, Responder, Scope};
+use ya_client_model::net::{Info, NET_API_V2_NET_PATH};
+
+use crate::error::Result;
+
+use super::client::ClientProxy;
+
+pub fn web_scope() -> Scope {
+    actix_web::web::scope(NET_API_V2_NET_PATH).service(get_info)
+}
+
+#[actix_web::get("/info")]
+async fn get_info() -> Result<impl Responder> {
+    //TODO do not create client every time
+    let client = ClientProxy::new().await?;
```

Comment:
This doesn't create Proxy every time, because internally there is `lazy_static` which returns the same instance.
But it would be better to pass it as `app_data`

Reply:
The split between hybrid and a central is a problem.
Above fn `pub fn web_scope() -> Scope` cannot be `async` which is required for `ClientProxy::new().await?`.
Going higher in call hierarchy I am ending outside of `hybrid` module so I cannot create net service `rest` method like this:
 `rest<CONTEXT: Provider<Self, ClientActor>>(ctx: &CONTEXT) -> actix_web::Scope`
 and also `ClientProxy` is neither public nor an Actor.